### PR TITLE
Plus Labs1-3 Initial

### PR DESCRIPTION
**Pull request by chrisakker**
Add Plus to labs 1-3.

### CODE REVIEW

File: Plus/labs/lab0/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Preparing your Computer for the NGINX Basics Workshop
+
+## Introduction
+
+In order to complete the Lab exercises in this Workshop on your own computer and not use F5 provided workshop environment, your computer should meet with the following license, hardware, software, and access requirements. If you are unable to meet these basic requirements, you will likely not be able to complete the Labs as written.  It is highly recommended that you have the minimum requirements.
```
Comment (apcurrier) [inline Plus/labs/lab0/prerequisites.md]: "and not use F5 provided" should be "and not use the F5 provided"

File: Plus/labs/lab0/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## Learning Objectives
+
+By the end of this Prerequisite instructions, you will be ready to do the Workshop lab exercises in your own system:
```
Comment (apcurrier) [inline Plus/labs/lab0/prerequisites.md]: sentence should end with "." not ":"

**Review by apcurrier — APPROVED** [review]

File: Plus/labs/lab0/prerequisites.md (hunk 1 above)
Comment (chrisakker) [reply Plus/labs/lab0/prerequisites.md]: fixed

File: Plus/labs/lab0/prerequisites.md (hunk 2 above)
Comment (chrisakker) [reply Plus/labs/lab0/prerequisites.md]: fixed

**Review by apcurrier — APPROVED** [review]

**Pull request merged**